### ISSUE TITLE
Changes to site configuration render longstanding kata unusable (aka old dog cannot learn new tricks)

### ISSUE THREAD

**Issue by paul-calvelage** [Codewars/codewars.com#268]
"You can't teach an old dog new tricks."  Neither can an author of a longstanding kata fix content issues that arise due to website configuration changes, apparently.

First, this is not a report of a content issue; rather, it is an issue with the way the website enforces the content editing process.  This has resulted in at least one longstanding kata being totally unusable.  (At first I thought this was merely a content issue, but after reviewing the discussion thread and previous content issue reports, it looks like the content issue cannot be fixed by the author due to the website issue I am reporting here.)

What has apparently happened in the case I discovered is this:
1. A kata was created, emerged from beta, etc.
2. The kata was completed by a good number of people. (1500+)
3. _**Changes to the site configuration rendered the kata unusable**_
4. Users discovered that the kata is unusable and reported the issues to the author.
5. The author informed the users that he is unable to correct the errors, because he no longer has access to the kata.
6. About a year after the first sign of trouble, my home page directed me to the kata and I discovered that the test cases will not execute.  At first, I thought this was merely a content issue, and was about to report it to the author.  However, after looking at the other issues reported the author claims to have no access to the kata.  I have no other prior relationship to the author other than as a user of the kata.

Somewhere the process has broken down, because I as a user was directed to a kata that has been broken for a year, with an author who is now active on the site willing to correct the issue if he only had access to do so.  No mechanism in the website apparently exists to resolve this.  That's the bug.

Looking at the steps leading up to this issue, I asked myself, "Could any longstanding kata be vulnerable to this bug, if a content issue comes up due to configuration changes and the author is denied access to fix the content issue?"  The answer is apparently yes, so the scope of this is beyond the content of the kata in question.

Once again to me, as a user, this looks like a bug in the website that leads to content issues in a longstanding and active kata being unable to be resolved by the author.  Why was I directed to a dead kata?  Can it be revived?  Where do kata go when they die?  Is there a way to filter out dead kata so I don't have this problem again?  It is possible that a policy decision may need to be made in order to resolve this bug.  

To emphasize, I, as a user do not care about the policy implications.  I am just noticing that my kata is broken, with no mechanism to either avoid or fix it.  The usual method of reporting the issue to the author seems to not be working since the author has no access.

Will this issue become more serious as more kata mature and die?  Can these "old dogs" adapt to new tricks / configuration changes?

The kata I discovered to be affected by this issue is www.codewars.com/kata/5255892719453db6d2000a23 (Part 1: Evil Programming Government Restrictions).  The discourse page of this kata has multiple issue reports starting from a year ago.  The author **is** responsive; however nothing can be done to correct the content bug, due to the website bug I am reporting here.

Perhaps other kata will be affected by this bug as well?

**Comment by jhoffner** [comment]
I agree, there is a gap in the process that needs to be filled. Its kind of both a bug and an enhancement for different reasons. One thing we could do is automatically open up the kata to allowing edits if more than maybe 3 or 4 issues are logged. Also the kata should be revalidated that its solution actually passes whenever a new issue is added. In this case that would have failed and then we could have automatically placed the kata back into beta.

**Comment by paul-calvelage** [comment]
Just noticed there were some ideas in #163 about this as well, so I am cross-referencing here.

**Comment by g964** [comment]
See #317, #320...

**Comment by jhoffner** [comment]
This issue is largely resolved with the recent 500 limit per-language change. We also now show issues within the kata header so users are aware that the kata may have problems. I'll leave this open as a reference for possible future enhancements but I think the severity of the issue has been minimized with recent updates.
